junit-jupiter-params/src/test/java/org/junit/jupiter/params/aggregator/AggregatorIntegrationTests.java

```diff
@@ -208,6 +208,17 @@ private EngineExecutionResults execute(DiscoverySelector... selectors) {
 		return EngineTestKit.execute("junit-jupiter", request().selectors(selectors).build());
 	}
 
+	@ParameterizedTest
+	@CsvSource({ "first", "second" })
+	void argumentsAccessorInvocationIndex(ArgumentsAccessor arguments) {
```

Comment:
Maybe this could be simplified by something like that:

```java
	@ParameterizedTest
	@CsvSource({ "1", "2", "3" })
	void argumentsAccessorInvocationIndex(ArgumentsAccessor arguments) {
		assertEquals(arguments.getInteger(0), arguments.getInvocationIndex());
	}
```

What do you think?

Reply:
This suggestion makes no sense to me, as there is no functional relationship between the numbers inside the CsvSource and the invocation index.

What do you think ?